evaluation_survey/static/js/src/evaluation_survey_edit.js

```diff
@@ -100,6 +100,7 @@ function EvaluationSurveyXBlockEdit(runtime, element) {
                 self.surveyId = payload.results[0].id
                 self.editVisibility(false);
                 self.state.status.main = STATUS_ENUM.CONNECTED;
+                $("#embedded_answers_edit_survey").children().filter((i,el)=>{return el.innerHTML===payload.results[0].name}).prop("selected", true)
```

Comment:
Ten select powinien być setowany tylko w newralgicznym przypadku opisywanym w zaadaniu. A nie za każdym razem gdy otrzymujesz zwrotkę. Druga rzecz, to dlaczego selectorem bezpośrednio nie wyszukujesz tych opcji, tylko przez children?

Reply:
No tak racja. Dodałem ograniczenie. Sprawdziłem i działa.